### PR TITLE
Inventory rbac v1

### PR DESCRIPTION
**Pull request by tahmidefaz**

### CODE REVIEW

File: configs/permissions/inventory.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "*": ["*", "read", "write"]
```
Comment (coderbydesign) [inline configs/permissions/inventory.json:2]: To clarify, you'll want the following permissions available for custom role creation, but you'll only create two roles, one for admin and one for viewer, but won't be using the write permission explicitly out of the gate, correct?
```
inventory:*:*
inventory:*:read
inventory:*:write
```

Comment (tahmidefaz) [reply configs/permissions/inventory.json:2]: Yes, I think that's what the team agreed on at this point. I'll update this PR if things change.

File: configs/permissions/inventory.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "*": ["*"],
+    "host_data": ["read", "write"]
```
Comment (jharting) [inline configs/permissions/inventory.json:3]: \<nitpick\/\> I think that the `_data` part is implicit i.e. a user can have read access to host_data, remediation_data, advisory_data, subscription_data, ... or we can just say read access to host, remediations, advisory, subscription, ... . I think the latter is cleaner and more in line with other apps.

File: configs/roles/inventory.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "roles": [
+    {
+      "name": "Inventory Administrator",
+      "description": "An Inventory Administrator role that grants read and write permissions to all Inventory resources.",
+      "system": true,
+      "platform_default": true,
+      "version": 1,
```
Comment (coderbydesign) [inline configs/roles/inventory.json:8]: Due to an idiosyncrasy in the way our seeding works, this needs to be started at `"version": 2`.
```suggestion
      "version": 2,
```

Comment (tahmidefaz) [reply configs/roles/inventory.json]: Thanks! Done! :+1:

**Pull request merged**